### PR TITLE
Update the configuration option for oc.

### PR DESCRIPTION
**Pull request by jackorp**

### CODE REVIEW

**Comment by pvalena** [comment]
LGTM.

File: deployment/openshift/about.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ That is okay. We will follow the suggestion on screen. The good news is that `oc
 
 To change the Docker configuration permanently, edit (as the `root` user) the file `/etc/docker/daemon.json` and add `172.30.0.0/16` to the key `insecure-registries`. For example:
 
-```JSON
```
Comment (pvalena) [inline deployment/openshift/about.md]: I'd drop this change. Or try something like YAML or Ruby :)) ..maybe it'll look better

**Comment by pvalena** [comment]
Otherwise LGTM. Feel free to merge if you disagree :).

File: deployment/openshift/about.md (hunk 1 above)
Comment (jackorp) [reply deployment/openshift/about.md]: Hah, figured out what the problem was, the Markdown formatted is allergic to any capitalized letters so JSON and Ruby all render out incorrectly, however `json` renders correctly :)

**Pull request merged**